neworigins/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
```

Comment:
we should delete all js/ts/json files

Reply:
when this is done
I committed map rendering to allow others see the map w/o client while development happens